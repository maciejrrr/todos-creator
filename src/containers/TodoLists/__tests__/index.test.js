import { TodoLists } from '../index';

describe('TodoLists', () => {
  describe('snapshots', () => {
    it('renders TodoLists', () => {
      const tree = renderer.create(<TodoLists />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });

  describe('enzyme', () => {
    it('has h1', () => {
      const component = shallow(<TodoLists />);
      expect(component.find('h1').length).toEqual(1);
    });
  });
});
