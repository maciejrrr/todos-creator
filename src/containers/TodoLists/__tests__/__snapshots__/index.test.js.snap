// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoLists snapshots renders TodoLists 1`] = `
<div>
  <h1>
    TodoLists
  </h1>
</div>
`;
